features/events/traps/blueprint-trapd-handler-minion.xml

```diff
@@ -0,0 +1,48 @@
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+	xmlns:ext="http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.1.0"
+	xsi:schemaLocation="
+		http://www.osgi.org/xmlns/blueprint/v1.0.0 
+		http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd
+
+		http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0
+		http://aries.apache.org/schemas/blueprint-cm/blueprint-cm-1.1.0.xsd
+
+		http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.1.0
+		http://aries.apache.org/schemas/blueprint-ext/blueprint-ext-1.1.xsd
+">
+
+	<cm:property-placeholder id="trapdNotificationHandlerMinionProperties" persistent-id="org.opennms.netmgt.trapd.handler.minion" update-strategy="none">
+		<cm:default-properties>
+			<cm:property name="brokerUri" value="tcp://127.0.0.1:61716" />
+		</cm:default-properties>
+	</cm:property-placeholder>
+
+	<bean id="trapdNotificationHandlerCamel" class="org.opennms.netmgt.trapd.TrapNotificationHandlerCamelImpl">
+		<argument value="seda:handleMessage" />
+	</bean>
+
+	<service interface="org.opennms.netmgt.trapd.TrapNotificationHandler" ref="trapdNotificationHandlerCamel" />
+
+
+	<bean id="activemq" class="org.apache.activemq.camel.component.ActiveMQComponent">
+		<!-- TODO: HZN-490 Add configurable ActiveMQ URI -->
+		<property name="brokerURL" value="${brokerUri}" />
+		<!-- TODO: HZN-490 Add configurable authentication -->
+		<!-- <property name="userName" value="karaf" /> <property name="password" 
+			value="karaf" /> -->
+	</bean>
+
+	<camelContext id="trapdNotificationHandlerMinion"
+		xmlns="http://camel.apache.org/schema/blueprint">
+		<propertyPlaceholder id="properties" location="blueprint:trapdNotificationHandlerMinionProperties" />
+		<route id="trapdHandlerMinion">
+			<from uri="seda:handleMessage" />
+			<convertBodyTo type="org.opennms.netmgt.trapd.TrapQueueProcessor" />
+			<!-- TODO: Use marshalling instead of serialization when transmitting object? -->
+			<!-- Broadcast message over ActiveMQ -->
+			<to uri="activemq:broadcastTrap?disableReplyTo=true" />
```

Comment:
Minion features should use the common queueing service instead of redefining their own activemq components

See https://github.com/OpenNMS/opennms/blob/develop/features/discovery/blueprint-discoverer.xml#L53